homeassistant/components/logbook.py

```diff
@@ -127,7 +129,6 @@ def as_dict(self):
 def humanify(events):
     """
     Generator that converts a list of events into Entry objects.
-
```

Comment:
Please add this one back. PEP257 states that documentation should have an empty line between the summary and the extended description.